sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -145,26 +145,28 @@ plot as shown below:
 Jupyter Notebooks
 =================
 
-At this point we should mention Jupyter (formerly IPython) notebooks.
```

Comment:
No research as far as I know. I was just condensing things, so I'll put this one back.